package.json

```diff
@@ -29,6 +29,7 @@
     "babel-plugin-transform-react-remove-prop-types": "0.4.24",
     "core-js": "3.2.1",
     "history": "4.10.1",
+    "lodash.isequal": "^4.5.0",
```

Comment:
for what

Reply:
I use it to create a deep equality selector, I could write my own function to check if they're equal